src/components/ProductForm/DeductiblesCeilingsTabForm.js

```diff
@@ -121,12 +120,18 @@ const DeductiblesCeilingsTabForm = (props) => {
             <TableRow className={classes.tableTitle}>
               <TableCell width="200" />
               {!isSplit && <TableCell>{formatMessage("DeductiblesCeilingsTabForm.all")}</TableCell>}
-              {isSplit && (
+              {isSplit && edited.ceilingInterpretation == "HEALTH_FACILITY_TYPE" && (
```

Comment:
Use '===' instead of '=='. Moreover, this "HEALTH_FACILITY_TYPE" should be moved to the constants.